services/uploads.js

```diff
@@ -83,13 +83,11 @@ Uploads.createUploadCredentials = function() {
  * See the [corresponding HTTP service documentation](https://docs.mapbox.com/api/maps/#create-an-upload).
  *
  * @param {Object} config
- * @param {string} config.mapId - The map ID to create or replace in the format `username.nameoftileset`.
+ * @param {string} config.tileset - The tileset ID to create or replace, in the format `username.nameoftileset`.
  *   Limited to 32 characters (only `-` and `_` special characters allowed; limit does not include username).
- * @param {string} config.url - Either of the following:
- *   - HTTPS URL of the S3 object provided by [`createUploadCredentials`](#createuploadcredentials)
- *   - The `mapbox://` URL of an existing dataset that you'd like to export to a tileset.
- *     This should be in the format `mapbox://datasets/{username}/{datasetId}`.
- * @param {string} [config.tilesetName] - Name for the tileset. Limited to 64 characters.
+ * @param {string} config.url - HTTPS URL of the S3 object provided by [`createUploadCredentials`](#createuploadcredentials)
+ * @param {string} config.name - Optional. The name of the tileset. Limited to 64 characters.
```

Comment:
```suggestion
 * @param {string} [config.name] - The name of the tileset. Limited to 64 characters.
```

use JSDoc to mark it as optional and then the generated docs will add optional automatically